v3/src/components/drag-drop/attribute-drag-overlay.scss

```diff
@@ -12,3 +12,7 @@
   justify-content: center;
   align-items: center;
 }
+
+.dnd-kit-drag-overlay {
+  pointer-events: none;
```

Comment:
A comment about this would be useful. Seems a bit counter-intuitive, but as long as regular dragging still works I have no objection.

Reply:
Normal dragging still works, but I'm honestly not totally sure this change didn't have other consequences. I made this change because the `WebViewDropOverlay` needs access to `mouseenter`, `mousemove`, and `mouseleave` events to send notifications, and the dnd overlay was apparently intercepting those events. Is there a less heavy-handed way to give `WebViewDropOverlay` access to the events it needs?